### PR TITLE
Call Recording Feature

### PR DESCRIPTION
**Pull request by jatinluthra14**

### CODE REVIEW

**Comment by garg-anuj** [comment]
@jatinluthra14 imports are missing in some classes

**Comment by jatinluthra14** [comment]
@garg-anuj Which are missing?

File: app/src/main/java/com/community/jboss/leadmanagement/CallReceiver.java [hunk 1]
```diff
@@ -7,9 +7,22 @@
...
+import com.aykuttasil.callrecord.CallRecord;
 
 public class CallReceiver extends BroadcastReceiver {
     private static final int ID = 47981;
+    private static String currentDateTimeString;
+
+    CallRecord callRecord = new CallRecord.Builder(this)
+            .setRecordFileName("Temp")            //Just temp name for initialisation
+            .setRecordDirName("LeadManagement_Recordings")
+            .setRecordDirPath(Environment.getExternalStorageDirectory().getPath()) // Save in External SDCard
```
Comment (garg-anuj) [inline app/src/main/java/com/community/jboss/leadmanagement/CallReceiver.java]: here `Environment`

File: app/src/main/java/com/community/jboss/leadmanagement/CallReceiver.java [hunk 2]
```diff
@@ -7,9 +7,22 @@
...
+            .setAudioEncoder(MediaRecorder.AudioEncoder.AMR_NB)
```
Comment (garg-anuj) [inline app/src/main/java/com/community/jboss/leadmanagement/CallReceiver.java]: `MediaRecorder`

Comment (jatinluthra14) [reply app/src/main/java/com/community/jboss/leadmanagement/CallReceiver.java]: Done

File: app/src/main/java/com/community/jboss/leadmanagement/CallReceiver.java [hunk 3]
```diff
@@ -7,9 +7,26 @@
...
+import android.os.Environment;
+import android.media.MediaRecorder;
+import java.text.SimpleDateFormat;
+import java.util.Date;
 
 public class CallReceiver extends BroadcastReceiver {
     private static final int ID = 47981;
+    private static String currentDateTimeString;
+
+    CallRecord callRecord = new CallRecord.Builder(this)
```
Comment (garg-anuj) [inline app/src/main/java/com/community/jboss/leadmanagement/CallReceiver.java]: `this` can not be available here.

Comment (garg-anuj) [reply app/src/main/java/com/community/jboss/leadmanagement/CallReceiver.java]: plus it need `this` to be context but in this class `this` will be an BroadcastReceiver

**Comment by jatinluthra14** [comment]
@garg-anuj I have removed .buildService() and kept .build() as .buildService() is no longer a method.
Build is successful too.

**Comment by jatinluthra14** [comment]
@garg-anuj What has to be done of it?

**Comment by garg-anuj** [comment]
Thanks for work

**Pull request closed without merging**